tests/test_scrubbers.py

```diff
@@ -38,3 +37,23 @@ def test_faker_scrubber_charfield(self):
 
         self.assertNotEqual(data.last_name, 'Foo')
         self.assertNotEqual(data.last_name, '')
+
+    def test_faker_scrubber_with_provider_arguments(self):
+        """
+        Use this as an example for Faker scrubbers with parameters passed along
+        """
+        data = DataFactory.create(ean8='8')
+        with self.settings(DEBUG=True, SCRUBBER_GLOBAL_SCRUBBERS={'ean8': scrubbers.Faker('ean', length=8)}):
+            call_command('scrub_data')
+        data.refresh_from_db()
+
+        # The EAN Faker will by default emit ean13, so this should fail if the parameter was ignored
+        self.assertEquals(8, len(data.ean8))
+
+        # Add a new scrubber for ean9
+        with self.settings(DEBUG=True, SCRUBBER_GLOBAL_SCRUBBERS={'ean9': scrubbers.Faker('ean', length=9)}):
+            call_command('scrub_data')
+        data.refresh_from_db()
+
+        # make sure it doesn't reuse the ean with length=8 scrubber
+        self.assertEquals(9, len(data.ean9))
```

Comment:
you probably mean `ean8` in these, right?

Reply:
> you probably mean `ean8` in these, right?

You're right. I mixed that up while replacing.
We can use both scrubbers on the same field and see if that changes the stuff, but I'm not sure if that works. Will try running that test again and fix the field names.

Reply:
Ok, it still works when using the same field for both, so I ditched the other one.